src/Files.App/UserControls/Widgets/WidgetsListControl.xaml

```diff
@@ -159,53 +159,35 @@
 		</Style>
 	</UserControl.Resources>
 
-	<Border>
```

Comment:
Instead of removing the border, might be better to remove the grid as border is supposedly a lighter control.

Reply:
Sure